c2cgeoportal/scripts/c2ctool.py

```diff
@@ -214,10 +214,29 @@ def get_project():
             return yaml.safe_load(f)
 
     def test_checkers(self):
-        http = httplib2.Http()
+        # Read project options for checker http session connection
+        try:
+            checker_protocol = self.project["checker_protocol"]
+        except KeyError:
+            checker_protocol = "http"
+            print("No checker_protocol defined in project, using default value {}".format(checker_protocol))
+        try:
+            checker_disable_ssl_certificate_validation \
+                = self.project["checker_disable_ssl_certificate_validation"]
```

Comment:
In 2.3 we never check the certificate => non needs to add an option for that.

Reply:
yes, but this is about 2.2.
So either we add this as an option for 2.2, or we deactivate the certificate check completely. In this PR it has already been implemented as an option, but I can remove the option and deactivate the certificate check everywhere in 2.2 if you prefer.

Reply:
+1 to deactivate the certificate check in any cases in 2.2 :-)

Reply:
But not every where, for the runtime call it should be enable by default.

Reply:
So I am disabling the certificate check in the c2ctool test_checkers (always; no configuration), but keeping the certificate check in the rest (configurable).